portiaui/app/components/spider-message.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+const { computed } = Ember;
+
+export default Ember.Component.extend({
+    api: Ember.inject.service(),
+    notificationManager: Ember.inject.service(),
+    hasSpider: computed.bool('currentSpider'),
+
+    actions: {
+        runSpider(spider) {
+            this.get('api').post('schedule', {
```

Comment:
I think it makes more sense to have a function in the `api` service which receives a spider and takes care of the interactions with the `notificationManager`.